packages/cli/src/commands/deploy/render.js

```diff
@@ -36,29 +34,35 @@ export const builder = (yargs) => {
     )
 }
 
-export const handler = async ({ side, prisma, dm: dataMigrate, serve }) => {
+export const handler = async ({ side, prisma, dm: dataMigrate }) => {
   const paths = getPaths()
   let commandSet = []
-  if (side == 'api') {
+  if (side === 'api') {
     if (prisma) {
       commandSet.push('yarn rw prisma migrate deploy')
     }
     if (dataMigrate) {
       commandSet.push('yarn rw dataMigrate up')
     }
-    if (serve) {
-      commandSet.push('yarn rw serve api')
-    }
-  } else if (side == 'web') {
+  } else if (side === 'web') {
     commandSet.push('yarn')
     commandSet.push('yarn rw build web')
   }
 
-  execa(commandSet.join(' && '), {
-    shell: true,
-    stdio: 'inherit',
-    cwd: paths.base,
-    extendEnv: true,
-    cleanup: true,
-  })
+  if (commandSet.length) {
+    await execa(commandSet.join(' && '), {
```

Comment:
What do you think of separating this out into different execa commands? This seems to depart from the way we use execa in most other places

Reply:
I'm not sure if we have a standard way of using Execa, do we? 😆

I don't think it's necessary here, but it couldn't hurt. I'll give it a go.

Reply:
@dac09 what do you think about these changes?
https://github.com/redwoodjs/redwood/pull/4278/commits/6bb82250b23efcfa9ef2fed6ca83808bd766a69e

Reply:
Looks great!